README.md

```diff
@@ -1073,6 +1057,25 @@ when you need to retrieve a single record by some attributes.
   = t '.title'
   ```
 
+* <a name="flat_localization"></a>
+  Use a flat structure in your localization files.
+<sup>[[link](#flat_localization)]</sup>
+
+  ```
+  # bad
+  en:
+    users:
+      name: 'Name'
+    accounts:
+      name: 'Name'
+    teams:
+      name: 'Name'
+
+  #good
+  en:
+    name: 'Name'
```

Comment:
bad example because there is no context in comparison to the nested one

suggestion: use user_name and account_name or team_name (or just user, account and team)